### PR TITLE
Customized Reusable Text Button UI

### PR DESCRIPTION
**Pull request by NadeenMedhat217**
- Created the UI of the Customized Text Button 
- Takes 2 props:
  1) One for the icon 
  2) Text that will be displayed on the Button 
image

### CODE REVIEW

File: coffee-shop/src/pages/Order/Order.jsx [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+import icons from "../../assets/icons/iconImport";
 
 const Order = () => {
   return (
     <div>
       Order page
       <CustomizedRadioButton />
+      <div className={styles.buttonContainer}>
+        <div className={styles.buttonWrapper}>
+          <CustomizedTextButton icon={icons.edit} text="Edit Address" />
```
Comment (arahman10783) [inline coffee-shop/src/pages/Order/Order.jsx]: the button needs `handleClick` props that carry the function that will be executed after clicking the button

**Review by arahman10783 — CHANGES_REQUESTED** [review]

File: coffee-shop/src/pages/Order/Order.jsx (hunk 1 above)
Comment (NadeenMedhat217) [reply coffee-shop/src/pages/Order/Order.jsx]: Done

File: coffee-shop/src/pages/Order/components/CustomizedTextButton/CustomizedTextButton.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import styles from "./CustomizedTextButton.module.css";
+
+const CustomizedTextButton = ({ icon, text, handleClick }) => (
```
Comment (arahman10783) [inline coffee-shop/src/pages/Order/components/CustomizedTextButton/CustomizedTextButton.js]: replace text props with label, it is more meeningfull

File: coffee-shop/src/pages/Order/Order.jsx [hunk 3]
```diff
@@ -1,20 +1,47 @@
...
-      <Button label="Order" />
       <DeliveryAddress
         location="Jl. Kpg Sutoyo"
         addressDetails="Kpg. Sutoyo No. 620, Bilzen, Tanjungbalai."
       />
+      <div className={styles.buttonContainer}>
+        <div className={styles.buttonWrapper}>
+          <CustomizedTextButton
+            icon={icons.edit}
+            text="Edit Address"
```
Comment (arahman10783) [inline coffee-shop/src/pages/Order/Order.jsx]: replace text props with label, it is more meeningfull

**Review by arahman10783 — APPROVED** [review]

**Review by arahman10783 — APPROVED** [review]

**Pull request merged**